docs/examples/plot_backtracking_sloped_terrain.py

```diff
@@ -0,0 +1,176 @@
+# from the azimuth of the panel orientation, which changes based on tracker
+# rotation angle. Because the tracker axis points in two directions, there are
+# two choices for the axis azimuth angle, and by convention (at least in the
+# northern hemisphere), the more southward angle is chosen:
+#
+# .. image:: ../_images/tracker_azimuth_angle_convention.png
+#   :alt: Image showing the azimuth convention for single-axis tracker arrays.
+#   :width: 500
+#   :align: center
+#
+# Note that, as with fixed-tilt arrays, the axis azimuth is determined as the
+# angle clockwise from north. The azimuth of the terrain's slope is also
+# determined as an angle clockwise from north, pointing in the direction
+# of falling slope. So for example, a hillside that slopes down to the east
+# has an azimuth of 90 degrees.
+#
+# Using the axis azimuth convention above, the sign convention for tracker
+# rotations is given by the
+# `right-hand rule <https://en.wikipedia.org/wiki/Right-hand_rule>`_.
+# Point the right hand thumb along the axis in the direction of the axis
+# azimuth and the fingers curl in the direction of positive rotation angle:
+#
+# .. image:: ../_images/tracker_rotation_angle_convention.png
+#   :alt: Image showing the rotation sign convention for single-axis trackers.
+#   :width: 500
+#   :align: center
+#
+# So for an array with ``axis_azimuth=180`` (tracker axis aligned perfectly
+# north-south), pointing the right-hand thumb along the axis azimuth has the
+# fingers curling towards the west, meaning rotations towards the west are
+# positive and rotations towards the east are negative.
+#
+# The ground slope itself is always positive, but the component of the slope
+# perpendicular to the tracker axes can be positive or negative. The convention
+# for the cross-axis slope angle follows the right-hand rule: align
+# the right-hand thumb along the tracker axis in the direction of the axis
+# azimuth and the fingers curl towards positive angles. So in this example,
+# with the axis azimuth coming out of the page, an east-facing, downward slope
+# is a negative rotation from horizontal:
+#
+# .. image:: ../_images/ground_slope_angle_convention.png
+#   :alt: Image showing the ground slope sign convention.
+#   :width: 500
+#   :align: center
+#
+
+# %%
+# Rotation curves
+# ---------------
+#
+# Now, let's plot the simple case where the tracker axes are at right angles
+# to the direction of the slope.  In this case, the cross-axis tilt angle
+# is the same as the slope of the terrain and the tracker axis itself is
+# horizontal.
+
+from pvlib import solarposition, tracking
+import pandas as pd
+import matplotlib.pyplot as plt
+
+# PV system parameters
+tz = 'US/Eastern'
+lat, lon = 40, -80
+gcr = 0.4
+
+# calculate the solar position
+times = pd.date_range('2019-01-01 06:00', '2019-01-01 18:00', closed='left',
+                      freq='1min', tz=tz)
+solpos = solarposition.get_solarposition(times, lat, lon)
```

Comment:
The rendered version has some weird syntax highlighting inconsistencies. I don't see anything wrong so probably safe to ignore.

Reply:
If you're talking about things being rendered blue-ish with clickable links, I think that's the intersphinx linking: https://sphinx-gallery.github.io/stable/configuration.html#add-intersphinx-links-to-your-examples

Future improvement: configure pvlib functions to link as well, or disable it for everything, for consistency.

Reply:
ohhh, cool, I didn't even realize they were links. I've recently visited some of the links but not all, so it's a mix of purple and blue. Ok to ignore.

<img width="706" alt="Screen Shot 2020-10-08 at 11 25 42 AM" src="https://user-images.githubusercontent.com/4383303/95499013-84390200-0959-11eb-8e60-8376403f0146.png">